components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -230,28 +230,29 @@ - (void)displaySnackbarViewForMessage:(MDCSnackbarMessage *)message {
                                     snackbarManager:self.manager];
   [self.delegate willPresentSnackbarWithMessageView:snackbarView];
   self.currentSnackbar = snackbarView;
-  self.overlayView.accessibilityViewIsModal =
-      self.manager.shouldEnableAccessibilityViewIsModal && ![self isSnackbarTransient:snackbarView];
+  self.overlayView.accessibilityViewIsModal = self.manager.shouldEnableAccessibilityViewIsModal;
   self.overlayView.hidden = NO;
   [self activateOverlay:self.overlayView];
 
   // Once the Snackbar has finished animating on screen, start the automatic dismiss timeout, but
   // only if the user isn't running VoiceOver.
+  __weak MDCSnackbarManagerInternal *weakSelf = self;
   [self.overlayView
       showSnackbarView:snackbarView
               animated:YES
             completion:^{
-              if ([self isSnackbarTransient:snackbarView]) {
+              __strong MDCSnackbarManagerInternal *strongSelf = weakSelf;
```

Comment:
Tip: `__strong` is the default (and I don't see you using it below). You can omit `__strong` modifiers on local variables like this. It would also be more consistent with the code below.

Reply:
Addressed in ca3870742a2cab331e3107faa76fb8868801ac82.